src/js/modules/Recollect.js

```diff
@@ -6,11 +6,11 @@ class Recollect extends Component {
   handleScriptLoad() {
     // Recollect is a third party script that isn't an import-able node module
     // rCw is the required container id to ensure styles are shown correctly
-    let loader = new window.Recollect.Widget.Loader({
-        area: "Austin",
-        name: this.props.name || 'calendar',
-        container: "#rCw"
-      });
+    const options = Object.assign({
```

Comment:
I'm interested to know what motivated this refactoring. Guessing it helps with multiple instances on a page. maybe?

Reply:
Yea, not multiple instances per page, but it was the simplest refactor, I thought, to enable adding a tabbed instance.